test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -15,3 +15,21 @@ Describe "Get-Process" -Tags "CI" {
         Get-Process | foreach-object { $_.Name | Should Not BeNullOrEmpty }
     }
 }
+
+Describe "Get-Process Formatting" -Tags "Feature" {
+    It "Should not have Handle in table format header" {
+        $types = "System.Diagnostics.Process","System.Diagnostics.Process#IncludeUserName"
+
+        foreach ($type in $types)
+        {
+            $formatData = Get-FormatData -TypeName $type -PowerShellVersion $PSVersionTable.PSVersion
+            $tableControls = $formatData.FormatViewDefinition | Where-Object {$_.Control -is "System.Management.Automation.TableControl"}
+            foreach ($tableControl in $tableControls)
+            {
+                $tableControl.Control.Headers.Label -match "Handle*" | Should BeNullOrEmpty
+                # verify that rows without headers isn't the handlecount (as PowerShell will create a header that matches the property name)
+                $tableControl.Control.Rows.Columns.DisplayEntry.Value -eq "HandleCount" | Should BeNullOrEmpty
+            }
+        }
+    }
+}
```

Comment:
I believe it is from the Unix world.
Some of MSFT require it. (git and GitHub mark it by red)
It would be good to have the `approved` tool to format code. Or at least a list of approved rules.
It's a bit confusing when one MSFT requires it, and other don't feel.